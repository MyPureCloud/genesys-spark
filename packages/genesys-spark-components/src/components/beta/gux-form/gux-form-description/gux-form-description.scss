@use '~genesys-spark/dist/beta/tokens/flare/global.scss';

::slotted(*) {
  margin: 0 !important;
  font-family: global.$gse-semantic-body-sm-regular-fontFamily !important;
  font-size: global.$gse-semantic-body-sm-regular-fontSize !important;
  font-weight: global.$gse-semantic-body-sm-regular-fontWeight !important;
  line-height: global.$gse-semantic-body-sm-regular-lineHeight !important;
  color: global.$gse-semantic-foreground-container-midEmphasis !important;
}
