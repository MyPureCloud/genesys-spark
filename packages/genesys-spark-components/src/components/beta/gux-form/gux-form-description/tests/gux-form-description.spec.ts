import { checkRenders } from '@test/specTestUtils';

import { GuxFormDescription } from '../gux-form-description';
import { renderConfig } from './gux-form-description.common';

const components = [GuxFormDescription];

describe('gux-form-description', () => {
  checkRenders([renderConfig], components);
});
