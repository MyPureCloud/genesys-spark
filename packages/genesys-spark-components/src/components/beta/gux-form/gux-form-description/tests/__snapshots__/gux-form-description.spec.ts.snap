// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`gux-form-description should render component as expected (1) 1`] = `
<gux-form-description>
  <template shadowrootmode="open">
    <slot>
    </slot>
  </template>
  <span>
    This is a simple paragraph to accompany the form header.
  </span>
</gux-form-description>
`;
